src/Workspaces/Core/Portable/FlowAnalysis/SymbolUsageAnalysis/SymbolUsageResult.cs

```diff
@@ -0,0 +1,90 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.FlowAnalysis.SymbolUsageAnalysis
+{
+    internal sealed class SymbolUsageResult
```

Comment:
do we need the ref-type here?  Looks like this could be a simple readonly struct.

Reply:
Sure, sure.

---
In reply to: [231672411](https://github.com/dotnet/roslyn/pull/31025#discussion_r231672411) [](ancestors = 231672411)